m3inference/m3twitter.py

```diff
@@ -265,3 +353,54 @@ def process_twitter(self, data):
             "output": pred[id]
         }
         return output
+
+
+    def process_twitter_batch(self, data_list, batch_size, num_workers):
```

Comment:
This function seems largely overlaps with the non-batched version. Could you call the non-batched version and then aggregate the result?